### DISCUSSION TITLE
[A/L] (2021/22) Funzioni - Esercizio 9

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#249] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152315302-610a650c-3fcf-47b3-b284-f21d12245a85.png)

**Comment by GattoBrizzolato** [comment]
![es9](https://user-images.githubusercontent.com/98970164/172479883-72ee2c84-8bc1-449e-8014-01637eb2f1bf.jpeg)
